### ISSUE TITLE
Is Multivariate Time based series forecasting in the roadmap for ML.Net?

### ISSUE THREAD

**Issue by HollandRisley** [dotnet/machinelearning#5638]
Is Multivariate Time based series forecasting in the roadmap for ML.Net?

**Comment by gareth-reid** [comment]
Yes I am very interested in this also. Any timeframe would be greatly appreciated.

**Comment by PrasOnWeb** [comment]
Please let us know if this is in the RoadMap?

**Comment by ericjohannsen** [comment]
Would be great to know whether/when you plan to support this.

**Comment by cedricouellet** [comment]
It has been a few years since the original request, any estimate on actual delivery of this feature? Would be very useful to see this in the future!

**Comment by superichmann** [comment]
?

**Comment by LameuleFR** [comment]
?
